frontend/src/locales/en.yml

```diff
@@ -219,6 +219,7 @@ task:
     compatibility: Compatibility
     connection: Connection
     migration-schema: Migration schema
+    schema-review: Schema review
```

Comment:
Also needs to add the translation to `./zh-CN.yml`

Reply:
Let's call this SQL review since it also applies to Data Change. Also this surfaces to the end-user, though our underlying model is called statement instead of SQL, the current specific use case is SQL, so let's use "SQL".

This is similar to other places that we surface "SQL" instead of "Statement" in the UI.